gtfs/__main__.py

```diff
@@ -1,25 +1,83 @@
 #!/usr/bin/env python
 """Command line interface for fetching GTFS."""
-import argparse
-import getpass
 import logging
-import sys
 
-from prettytable import PrettyTable
+import typer
+from chalky import chain
+from halo import Halo
+from prettytable.colortable import ColorTable, Themes
+from typing_extensions import Annotated
 
-from gtfs.utils.FeedSource import FeedSource
-import feed_sources
-# import all the available feed sources
-# pylint: disable=I0011,wildcard-import
-from feed_sources import *
+from .feed_source import FeedSource
+from .feed_sources import __all__ as feed_sources
+from .utils.constants import Predicate
+from .utils.geom import bbox_contains_bbox, bbox_intersects_bbox
 
 logging.basicConfig()
 LOG = logging.getLogger()
 LOG.setLevel(logging.INFO)
+app = typer.Typer()
 
-def fetch_all(sources=None):
-    """Fetch from all FeedSources in the feed_sources directory.
 
+def check_bbox(bbox: str):
+    if "," not in bbox or len(bbox.split(",")) != 4:
```

Comment:
```suggestion
    if len(bbox.split(",")) != 4:
```

it'll implicitly check the first condition as well